--- influxdb.conf.orig	2021-04-20 00:55:19.000000000 +0300
+++ influxdb.conf	2021-05-04 20:05:02.411447680 +0300
@@ -9,7 +9,7 @@
 # The data includes a random ID, os, arch, version, the number of series and other
 # usage data. No data from user databases is ever transmitted.
 # Change this option to true to disable reporting.
-# reporting-disabled = false
+reporting-disabled = true
 
 # Bind address to use for the RPC service for backup and restore.
 # bind-address = "127.0.0.1:8088"
@@ -257,10 +257,10 @@
   # flux-log-enabled = false
 
   # The bind address used by the HTTP service.
-  # bind-address = ":8086"
+  bind-address = "127.0.0.1:{{ influxdb_port }}"
 
   # Determines whether user authentication is enabled over HTTP/HTTPS.
-  # auth-enabled = false
+  auth-enabled = true
 
   # The default realm sent back when issuing a basic auth challenge.
   # realm = "InfluxDB"
@@ -325,10 +325,10 @@
   # max-connection-limit = 0
 
   # Enable http service over unix domain socket
-  # unix-socket-enabled = false
+  unix-socket-enabled = true
 
   # The path of the unix domain socket.
-  # bind-socket = "/var/run/influxdb.sock"
+  bind-socket = "/var/lib/influxdb/influxdb.sock"
 
   # The maximum size of a client request body, in bytes. Setting this value to 0 disables the limit.
   # max-body-size = 25000000
